client/platform/desktop/frontend/components/Export.vue

```diff
@@ -183,6 +187,16 @@ export default defineComponent({
               </span>
             </div>
           </template>
+
+          <template v-if="checkedTypes.length">
+            <v-checkbox
+              v-model="data.excludeUncheckedTypes"
+              label="export checked types only"
```

Comment:
Maybe add a `details` prop with `Export only the types currently enabled in the type filter`.  Get rid of `hide-details`.

I think the extra explanation is needed.